dev-python/pyilmbase/pyilmbase-2.2.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 cmake-utils
+
+DESCRIPTION="IlmBase Python bindings"
+HOMEPAGE="http://www.openexr.com"
+SRC_URI="http://download.savannah.gnu.org/releases/openexr/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0/23" # based on SONAME
+KEYWORDS="~x86 ~amd64"
+
```

Comment:
I think replacing this empty line with `IUSE=""` would make the `REQUIRED_USE` look less out of place. At least I won't be looking for misplaced `IUSE`.

Reply:
Okay.